integration-test.sh

```diff
@@ -15,12 +15,11 @@ export MINIMIZE_DOWNTIME=0
 
 if [[ "$test_option" == "--initial-install" ]]; then
```

Comment:
The goal is to rewrite this file entirely in python using pytest, but baby steps!